doc/source/user_guide/numpy_images.rst

```diff
@@ -33,14 +33,14 @@ types and how ``scikit-image`` treats them.
 NumPy indexing
 --------------
 
-NumPy indexing can be used both for looking at the pixel values and to
-modify them::
+NumPy indexing (0-based!) can be used both for looking at the pixel values
```

Comment:
This exclamation mark only makes sense to some people... How about the following:

```
NumPy indexing can be used to look at pixels values::

    >>> # get the pixel at row 10 and column 20
    >>> camera[10, 20]

As a reminder, NumPy indexing, like Python indexing, is 0-based (starts at 0)::

    >>> # Get the value at pixel at 0th row and 10th column
    >>> camera[0, 10]
```

Reply:
@jni I appreciate this formulation, being less ambiguous